MIDIchlorians/MIDIchlorians/GridController.swift

```diff
@@ -16,7 +16,7 @@ protocol PadDelegate: class {
 // Manages the currently visible grid of pads using a GridCollectionView
 class GridController: NSObject {
     var view: UIView
-    var gridView: UICollectionView {
```

Comment:
@ngzhian Is there a reason why this was `UICollectionView`? Is it okay if I change it to `GridCollectionView`? I need it to be `GridCollectionView` so that it implements `AnimatableGrid` protocol, and changing it to `GridCollectionView` doesn't seem to break anything..